apps/prairielearn/src/lib/aws.ts

```diff
@@ -115,7 +135,10 @@ export async function downloadFromS3(s3Bucket, s3Path, localPath, options = {})
     Bucket: s3Bucket,
     Key: s3Path,
   });
-  const s3Stream = /** @type {import('stream').Readable} */ (res.Body);
+  if (res.Body == null) {
+    throw new Error('No data returned from S3');
+  }
+  const s3Stream = res.Body as Readable; /* StreamingBlobPayloadOutputTypes -> Readable ? */
```

Comment:
🙄 https://github.com/aws/aws-sdk-js-v3/issues/4720 - why can't AWS make this nicer?

There's a suggestion here that might work better: https://github.com/aws/aws-sdk-js-v3/issues/4720#issuecomment-1773655119